packages/react-charting/src/components/CommonComponents/CartesianChart.styles.ts

```diff
@@ -138,18 +143,33 @@ export const getStyles = (props: ICartesianChartStyleProps): ICartesianChartStyl
     calloutlegendText: {
       ...fonts.small,
       lineHeight: '16px',
+      selectors: {
+        [HighContrastSelector]: {
+          color: 'rgb(255, 255, 255)',
```

Comment:
Could you explain this rgb 255 color value. Why do you need to hardcode.

Reply:
The color of the text in the tool tip was getting very dim after applying the high contrast theme so to make it brighter i did that. Should i use the theme instead.

Reply:
Is the tool tip text customizable by the user? If yes it should be taken from props.